traceevents/src/main/java/com/tomtom/kotlin/traceevents/TraceEvent.kt

```diff
@@ -64,7 +80,7 @@ data class TraceEvent(
         if (stackTraceHolder != other.stackTraceHolder) return false
         if (eventName != other.eventName) return false
         if (!args.contentDeepEquals(other.args)) return false
-
+        if (!parameterNames.contentEquals(other.parameterNames)) return false
```

Comment:
Is there any significance in the difference between `contentEquals` and the `contentDeepEquals` on the line above?

Reply:
`parameterNames` is an array of `String` so `contentDeepEquals` doesn't do anything on top of `contentEquals`.